posturebest/Views/Torso3DView.swift

```diff
@@ -0,0 +1,78 @@
+//
+//  Torso3DView.swift
+//  posturebest
+//
+//  Created by Madeline Coco on 10/16/24.
+//
+
+import SwiftUI
+import SceneKit
+
+struct Model3DView: UIViewRepresentable {
+    func makeUIView(context: Context) -> SCNView {
+        let sceneView = SCNView()
+        sceneView.allowsCameraControl = false
+        sceneView.showsStatistics = true
+        sceneView.backgroundColor = UIColor.white
+        
+        // Create and set up the scene
+        let scene = SCNScene()
+        sceneView.scene = scene
+        
+        loadModel(into: scene)
+        
+        addLights(to: scene)
+        
+        addCamera(to: scene)
+        
+        return sceneView
+    }
+    
+    func updateUIView(_ uiView: SCNView, context: Context) {
+        // Update the view if needed
+    }
+    
+    private func loadModel(into scene: SCNScene) {
+        guard let modelScene = SCNScene(named: "human_woman.dae") else {
+            print("Failed to load the model.")
+            return
+        }
+        
+        
+        let modelNode = modelScene.rootNode
+            modelNode.scale = SCNVector3(0.1, 0.1, 0.1) // Adjust scale as needed
+            scene.rootNode.addChildNode(modelNode)
+        
+        // Adjust the camera position to fit the model
+            fitCamera(to: modelNode, in: scene)
+            print("Model loaded successfully!")
+        
+    }
+    
+    private func addLights(to scene: SCNScene) {
+        // Ambient light
+        let ambientLight = SCNLight()
+        ambientLight.type = .ambient
+        ambientLight.color = UIColor.white
+        let ambientLightNode = SCNNode()
+        ambientLightNode.light = ambientLight
+        ambientLightNode.position = SCNVector3(0, 10, 0)
+        scene.rootNode.addChildNode(ambientLightNode)
+        
+        // Directional light
```

Comment:
I understand wanting to leave comments in but i think these code blocks are self explanatory. The comments don't add anything that we dont already know (for ex, these vars are already called `directionalLight`

Reply:
I think if you wanted to keep them in, we could make them more informative, like why we need directional light/ what its doing or something like that